scripts/actions/check-job-progress.js

```diff
@@ -74,7 +72,6 @@ const getBatchStatus = (accessToken) => async ({ batchUid, jobId }) => {
     const jobData = await vendorRequest({
       method: 'GET',
       endpoint: `/jobs-api/v3/projects/${PROJECT_ID}/jobs/${translationJobUid}`,
```

Comment:
Do we not need any authorization for these api calls?

Reply:
we do. but now that specific concern is handled entirely inside of vendorRequest.

Reply:
ahhh right ye that makes sense to me noooow